code/modules/client/preferences.dm

```diff
@@ -1449,12 +1436,6 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 					if(new_tail)
 						features["tail_lizard"] = new_tail
 
-				if("tail_human")
```

Comment:
Does this not remove the option to choose your tail as felinid?

Reply:
the only tail choice that felinids got was "cat tail" or "no tail". this pref is pointless